assets/src/common/components/featured-image-select-media-frame.js

```diff
@@ -1,125 +0,0 @@
-/**
```

Comment:
This is mainly moved to [select-media-frame.js](https://github.com/ampproject/amp-wp/pull/2737/files#diff-507fa4419fd99ea17660e963ced76788).